### PR TITLE
Change behavior of the problem_recorder and driver_recorder options on dashboard command

### LINKED ISSUE #318: Change behavior of the problem_recorder and driver_recorder options on dashboard command
### Desired capability or behavior.

Currently the dashboard command options: problem_recorder and driver_recorder are used to indicate if those files should be read and used for the creation of various tabs on the dashboard. 

Feedback indicates that users would prefer if the dashboard always used those case recorder files and that these options be used to indicate that a non-standard name for the file was used. The exception to that is if driver_recorder is set to None, which would indicate not reading the driver recorder file, which could be very large and the resulting dashboard tab not of interest to the user. 

This will mean that the file names used for the functions run_aviary and run_aviary_problem need to be changed to match the defaults used by the dashboard command code. 

### Is your feature request related to a problem? Please describe.

_No response_

### Associated Bug Report

_No response_

### PR DESCRIPTION
**Pull request by hschilling**
### Summary

Modified the way the dashboard command works so that the dashboard always uses the problem and driver case recorder files and that the problem_recorder and driver_recorder options be used to indicate that a non-standard name for the file was used. The exception to that is if driver_recorder is set to None, which would indicate not reading the driver recorder file, which could be very large and the resulting dashboard tab not of interest to the user.

Also changed the name of the default filenames to be `problem_history.db` and `driver_history.db`. 

### Related Issues

- Resolves #318 

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: aviary/visualization/dashboard.py [hunk 1]
```diff
@@ -436,8 +455,8 @@ def dashboard(script_name, problem_recorder, driver_recorder, port):
         Name of the script file whose results will be displayed by this dashboard.
     problem_recorder : str
         Name of the recorder file containing the Problem cases.
-    driver_recorder : str
-        Name of the recorder file containing the Driver cases.
+    driver_recorder : str or None
+        Name of the recorder file containing the Driver cases. If None, ignore this file
```
Comment (crecine) [inline aviary/visualization/dashboard.py:459]: I'd prefer something more descriptive than "ignored", maybe something like:
```suggestion
        Name of the recorder file containing the Driver cases. If None, the driver tab will not be added
```

**Review by crecine — APPROVED** [review]

File: aviary/visualization/dashboard.py (hunk 1 above)
Comment (hschilling) [reply aviary/visualization/dashboard.py]: Good suggestion! Fixed

**Comment by crecine** [comment]
https://github.com/OpenMDAO/Aviary/pull/330 changed aviary/docs/user_guide/outputs_and_how_to_read_them from .md to .ipynb, if you need help resolving the merge conflict, let me know.

**Review by Kenneth-T-Moore — APPROVED** [review]

**Comment by crecine** [comment]
@hschilling Just created a PR to your branch to resolve the merge conflict, which should fix the doc failure

**Comment by hschilling** [comment]
Thanks!

**Comment by crecine** [comment]
> Thanks!
You'll have to approve it here
https://github.com/hschilling/om-Aviary/pull/1

**Pull request merged**